shuup/core/models/_orders.py

```diff
@@ -456,7 +457,7 @@ def create_payment(self, amount, payment_identifier=None, description=''):
         payments = self.payments.order_by('created_on')
 
         total_paid_amount = self.get_total_paid_amount()
-        if total_paid_amount >= self.taxful_total_price.amount:
+        if total_paid_amount >= self.taxful_total_price.amount and not self.is_zero_price():
```

Comment:
`if total_paid_amount >= self.taxful_total_price.amount and self.taxful_total_price:` would have worked as well. I don't think `is_zero_price` is really that useful method for `Order`.